src/gc/gcinterface.h

```diff
@@ -537,6 +537,14 @@ class IGCHeap {
     ===========================================================================
     */
 
+    virtual int GetTimeInGC() = 0;
```

Comment:
>irtual int GetTimeInGC() = 0; [](start = 5, length = 29)

these are not for hosting APIs - they should belong in the APIs for BCL section, like GetLastGCStartTime.

also the names are very ambiguous - GetTotalAllocation for what period of time? these should be renamed to GetLastGCTimeInGC/GetLastGCGenerationSize/etc...following the GetLastGCStartTime convention

Reply:
Thanks - I renamed GetTimeInGC and GetGenerationSize to GetLastGCTimeInGC and GetLastGCGenerationSize. Something I wanted to clarify was GetTotalAllocation - the current implementation is that GetTotalAllocation just returns an ever-increasing number which represents the total allocated bytes so far. The runtime counter then turns this into a rate so the end user would only see the total allocated bytes since it (the runtime counter) last reported the value. Were you suggesting that I change this to return the total allocated bytes during the last GC and then rename it to something like GetLastGCTotalAllocation?